test/specs/mainnet/swap/Smoke_CrossChainQuote.spec.js

```diff
@@ -0,0 +1,4195 @@
+        "The value of the estimatedGasPrice field of the Submit Batch Response is not displayed."
+      );
+      EstimatedGasPrice_Submit = submissionResponse.estimatedGasPrice._hex;
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.strictEqual(
+        EstimatedGasPrice_Estimate,
+        EstimatedGasPrice_Submit,
+        "The Estimated Gas Price value is not displayed correctly."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNull(
+        submissionResponse.feeToken,
+        "It is not expected behaviour of the feeToken in the Submit Batch Response."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNotEmpty(
+        submissionResponse.feeAmount._hex,
+        "The value of the feeAmount field of the Submit Batch Response is not displayed."
+      );
+      FeeAmount_Submit = submissionResponse.feeAmount._hex;
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.strictEqual(
+        FeeAmount_Estimate,
+        FeeAmount_Submit,
+        "The Fee Amount value is not displayed correctly."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNotEmpty(
+        submissionResponse.feeData,
+        "The value of the feeData field of the Submit Batch Response is not displayed."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNull(
+        submissionResponse.delayedUntil,
+        "It is not expected behaviour of the delayedUntil in the Submit Batch Response."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+  });
+
+  // CROSS CHAIN QUOTES ON BSC NETWORK
+  it("Setup the SDK for Bsc network and perform the cross chain quote action", async () => {
+    // initialize the sdk
+    sdkMainNet = new Sdk(process.env.PRIVATE_KEY, {
+      env: EnvNames.MainNets,
+      networkName: NetworkNames.Bsc,
+    });
+
+    assert.strictEqual(
+      sdkMainNet.state.accountAddress,
+      "0xa5494Ed2eB09F37b4b0526a8e4789565c226C84f",
+      "The EOA Address is not displayed correctly."
+    );
+
+    // Compute the smart wallet address
+    let smartWalletOutput = await sdkMainNet.computeContractAccount();
+    smartWalletAddress = smartWalletOutput.address;
+    console.log("Smart wallet address: ", smartWalletAddress);
+
+    assert.strictEqual(
+      smartWalletAddress,
+      "0x666E17ad27fB620D7519477f3b33d809775d65Fe",
+      "The smart address is not displayed correctly."
+    );
+
+    let BscUSDC = "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d"; // Bsc - USDC
+    let MaticUSDC = "0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174"; // Matic - USDC
+
+    let fromChainId = NETWORK_NAME_TO_CHAIN_ID[NetworkNames.Bsc];
+    let toChainId = NETWORK_NAME_TO_CHAIN_ID[NetworkNames.Matic];
+    let fromTokenAddress = BscUSDC;
+    let toTokenAddress = MaticUSDC;
+    let fromAmount = ethers.utils.parseUnits("0.5", 6);
```

Comment:
Remember to lower this amount as we will burn through our test funds

Reply:
Yeah, We need to discuss the same with the help of Vignesh.